frontend/test/metabase/scenarios/question/filter.cy.spec.js

```diff
@@ -17,70 +17,203 @@ describe("scenarios > question > filter", () => {
     signInAsAdmin();
   });
 
-  it.skip("should load needed data (metabase#12985)", () => {
-    // Save a Question
-    openProductsTable();
-    cy.findByText("Save").click();
-    cy.findByPlaceholderText("What is the name of your card?")
-      .clear()
-      .type("Q1");
-    cy.findAllByText("Save")
-      .last()
-      .click();
-    cy.findByText("Not now").click();
-
-    // From Q1, save Q2
-    cy.visit("/question/new");
-    cy.findByText("Simple question").click();
-    cy.findByText("Saved Questions").click();
-    cy.findByText("Q1").click();
-    cy.findByText("Save").click();
-    cy.findByPlaceholderText("What is the name of your card?")
-      .clear()
-      .type("Q2");
-    cy.findAllByText("Save")
-      .last()
-      .click();
+  describe("dashboard filter dropdown/search (metabase#12985)", () => {
+    it("Repro 1: should work for saved nested questions", () => {
+      cy.log("**-- 1. Create base card --**");
+
+      cy.request("POST", "/api/card", {
+        name: "Q1",
+        dataset_query: {
+          database: 1,
+          query: { "source-table": PRODUCTS_ID },
+          type: "query",
+        },
+        display: "table",
+        visualization_settings: {},
+      }).then(({ body: { id: Q1_ID } }) => {
+        cy.log("**-- 2. Create nested card based on the first one --**");
+
+        cy.request("POST", "/api/card", {
+          name: "Q2",
+          dataset_query: {
+            database: 1,
+            query: { "source-table": `card__${Q1_ID}` },
+            type: "query",
+          },
+          display: "table",
+          visualization_settings: {},
+        }).then(({ body: { id: Q2_ID } }) => {
+          cy.log("**-- 3. Create a dashboard --**");
+
+          cy.request("POST", "/api/dashboard", {
+            name: "12985D",
+          }).then(({ body: { id: DASHBOARD_ID } }) => {
+            cy.log("**-- 4. Add 2 filters to the dashboard --**");
+
+            cy.request("PUT", `/api/dashboard/${DASHBOARD_ID}`, {
+              parameters: [
+                {
+                  name: "Date Filter",
+                  slug: "date_filter",
+                  id: "78d4ba0b",
+                  type: "date/all-options",
+                },
```

Comment:
Is this part in this or other tests? Otherwise we should remove it to simplify the test.

Reply:
This was your original repro for v1, step 5:
> 5. Add two filter linked to Q2, a time "All options" for "Created At" and a "Other categories" for "Category" and save dashboard

Reply:
Okay, then I'll create another issue for that.